addons/pos_self_order/controllers/webmanifest.py

```diff
@@ -0,0 +1,38 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import mimetypes
+import re
+
+from odoo import http
+from odoo.http import request
+from odoo.addons.web.controllers.webmanifest import WebManifest as WebWebManifest
+
+
+class WebManifest(WebWebManifest):
+    def _get_scoped_app_name(self, app_id):
+        if app_id == "pos_self_order":
+            res_id = re.findall(r'pos-self/(\d+)', request.params['path'])[0]
+            record = request.env['pos.config'].search([('id', '=', res_id)])
+            if record:
+                return record.name
```

Comment:
```suggestion
            if match := re.findall(r'pos-self/(\d+)', request.params['path'])
                if record := request.env['pos.config'].search([('id', '=', match[0])])
                    return record.name
```

Reply:
Also it is **very** ugly to use request.params. I would definitely prefer that it takes the path as function argument. I see that you did `_path = unquote(path)` in your controller, that change didn't reflected inside the `request.params` dictionary.